Userland/Applications/DisplaySettings/main.cpp

```diff
@@ -27,6 +28,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto app_icon = GUI::Icon::default_icon("app-display-settings");
 
     auto window = TRY(GUI::SettingsWindow::create("Display Settings"));
+    (void)TRY(window->add_tab<DisplaySettings::ThemesSettingsWidget>("Themes"));
```

Comment:
Please put the themes tab after the background tab :^)

Reply:
Done :)